canonical-web-and-design/react-components
# Remove form from searchbox

## Done

- Removed the form wrapping the `SearchBox` component, and replaced it with a div.
- Also removed the `onSubmit` prop, since it seems like there's no reason to have one without a form, and added a `submittable` prop in its place, that renders the submit button if it's ever needed.

## QA

### Storybook

To see rendered examples of all react-components, run:

```shell
yarn start
```

### QA in your project

from `react-components` run:

```shell
yarn build
npm pack
```

Install the resulting tarball in your project with:

```shell
yarn add <path-to-tarball>
```

### QA steps

- Visit ?path=/docs/searchbox--external-state#submittable
- See that it looks good, and the docs make sense

## Fixes

Fixes: #566  .
